scenes/preload.ts

```diff
@@ -169,6 +169,8 @@ export class PreloadScene extends Phaser.Scene {
             margin: 1
         });
         this.load.spritesheet("icon-item-set", "assets/images-extruded/items/icon-item-set.png", tilesetConfig);
+        this.load.spritesheet("price-tag-icons", "assets/images/tilesets/price-tag-icons.png", tilesetConfig);
```

Comment:
Make sure that asset license allows us to use it. If necessary, credit the author in ./credits.md. If you want, you can add yourself there too :-)

Reply:
Thank you for reminding me of that. I have made all assets except the copper coin and will credit the authors in ./credits.md